libs/features/mas/web-components/src/variants/special-offer.js

```diff
@@ -42,6 +42,10 @@ export class SpecialOffer extends VariantLayout {
     :host([variant='special-offers']) {
       min-height: 439px;
     }
+
+    :host([variant='special-offers']) {
```

Comment:
this change is needed to render a merch-card with valid dimension without the responsive grid.